tests/test_logic.py

```diff
@@ -0,0 +1,40 @@
+from context import core
+from context import models
+
+from models import model
+
+from core import logic
+
+import unittest
+
+class test_create_room(unittest.TestCase):
+    def setUp(self):
+        self.white_char_in_name = logic.create_room('office', "name        ")
+        self.white_char_in_typr = logic.create_room('livingspace        ', "name")
+
+
+    def test_create_room_office(self):
```

Comment:
The function create_room doesn't return an instance of a room created. Neither does it's helper. I'm worried that the tests differ from the implementation